CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Ansible Modules for HPE OneView Change Log
 
+## v5.3.1
+
+This release is to support resource name instead of resource URIs in server profile and profile template.
+
+#### Major changes
+ 1. Updated ReplaceServerProfileNamebyUris class to support the schema changes in latest API version.
+ 2. Added an end-to-end example playbook for server profile tempalte with the below resources
```

Comment:
Can you please correct this typo - Template*

Reply:
Done.